test/unit/express-tooling.ts

```diff
@@ -30,7 +30,7 @@ describe('express tooling', () => {
     next = sinon.spy();
 
     responseMock = {};
-    responseMock.sendStatus = sinon.spy((status: string) => {});
+    responseMock.sendStatus = sinon.spy((status: string) => ({}));
```

Comment:
A suggestion for a discussion here @ISO50 @Retro64 I find it easier to follow the tests if all the "given state" is actually defined in the test case itself. I know that we often use `before` and `beforeEach` and I see the use case, but here I do not really see the need for it and it just seems to be cleaner if the declaration of `next` and `responseMock` would be in the test cases since that are actually the things we test. 

I am even wondering whether it would be better to also move the declaration of `requestMock` into the test cases. But there I do not see such a need cause we do no tests on this object, its purely for mocking and to keep things running as far as I see it. What do you think?

Reply:
Maybe as part of #132 as we decided to unify tests within this issue?